src/clipboard-action.js

```diff
@@ -140,8 +140,8 @@ class ClipboardAction {
      * Sets the `action` to be performed which can be either 'copy' or 'cut'.
      * @param {String} action
      */
-    set action(action) {
-        this._action = action || 'copy';
+    set action(action = 'copy') {
+        this._action = action;
```

Comment:
I believe that is safer to use the 'or' operator here because it'll work if the passed parameter is an empty string.